src/modules/editor/ScriptEditorImplementation.cpp

```diff
@@ -67,6 +67,7 @@ extern std::set<ScriptEditorImplementation *> g_pScriptEditorWindowList;
 extern KviModule * g_pEditorModulePointer;
 
 static QColor g_clrBackground(0, 0, 0);
+static QColor g_clrBackgroundHighlight(50, 50, 50);
```

Comment:
This doesn't force them into using a dark theme.  The default theme for the script editor is a dark theme, so the default for "Background highlight" should be for a dark theme as well.  It doesn't make sense to default it to match a light theme as the user would have had to change it from dark to something else.

I thought about doing `lighten(##)` but that would only make sense for dark themes and it would fail on light themes, hence just setting it as something that makes sense for default and allowing the pro themers to change it to what they want.  Maybe they want a pink highlight on a normally black background?